inference-engine/tests/ie_test_utils/functional_test_utils/network_utils.cpp

```diff
@@ -159,6 +159,8 @@ namespace FuncTestUtils {
                 }
             } else {
                 if (item.first == "originalLayersNames") continue;
+                // pooled_h and pooled_w are output_size for ROIPooling
```

Comment:
The comment does not explain why the code was modified. Could you provide more detailed explanation?

Reply:
Sure, the specification says that there should be two parameters- _pooled_h_ and _pooled_w_, our implementation of this op has a single parameter _output_size_, we shouldn't be failing just if this parameter is not present.

You can take a look at the single commit I've added just for _roi pooling_ to maybe get a better perspective on what changes I made to this operator.

Reply:
Please, add this comment to the code itself :)